app/styles/utilities/animation.scss

```diff
@@ -1,8 +1,11 @@
-.fade {
-  opacity: 0;
-
-  .in {
+@keyframes in {
+  from {
+    opacity: 0;
+  }
+  to {
```

Comment:
These keyframes add a pretty nice effect. In my opinion, I really think we should with an effect similar to what [this website](https://devart.withgoogle.com/) has. Try clicking on a language and watch the list pull its old elements in and then push its new ones out. I think its pretty elegant. What do you think?